.openshift-ci/jobs/push-images/push-builder-image.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+export PROJECT_DIR=/go/src/github.com/stackrox/collector
+cd "$PROJECT_DIR"
+
+collector_version="$(make tag)"
+export COLLECTOR_VERSION="$collector_version"
+
+export PUBLIC_REPO=quay.io/stackrox-io
+export QUAY_REPO=quay.io/rhacs-eng
+
+shopt -s nullglob
+for cred in /tmp/secret/**/[A-Z]*; do
+    export "$(basename "$cred")"="$(cat "$cred")"
+done
+
+# shellcheck source=SCRIPTDIR/registry_rw_login.sh
+source "${PROJECT_DIR}/.openshift-ci/jobs/push-images/registry_rw_login.sh"
+
+image_repos=(
+    "${QUAY_REPO}"
+    "${PUBLIC_REPO}"
+)
+
+oc registry login
+
+for repo in "${image_repos[@]}"; do
+    registry_rw_login "$repo"
+
+    image="${repo}/collector-builder:${collector_version}"
+    echo "Pushing image ${image}"
+    oc image mirror "${COLLECTOR_BUILDER}" "${image}"
```

Comment:
Is a separate job script needed? We could also add to push-images/push-images.sh?

Reply:
Agreed. I have combined push-builder-image.sh and push-images.sh.